pipeline_steps/pubcloud.groovy

```diff
@@ -202,9 +202,20 @@ def runonpubcloud(Map args=[:], body){
       try {
         delPubCloudSlave(args)
       } catch (e){
-        print "Error while cleaning up, swallowing this exception to prevent "\
-              +"cleanup errors from failing the build: ${e}"
-        common.create_jira_issue("RE", "Cleanup Failure: ${env.BUILD_TAG}")
+        // Ideally timeouts would be within the node block so that clenaup
+        // cannot fail due to timeout. However that is more complicated
+        // when the node block is reused in different situations (eg
+        // runonpubcloud).
+        // As we are planning to move to node pool, its not worth investing
+        // the time to refactor that, so drop timeout exceptions instead
+        // of creating useless jira issues.
+        if(e.getClass().name == "org.jenkinsci.plugins.workflow.steps.FlowInterruptedException"){
+          print "Caught timeout exception, dropping"
```

Comment:
Better to just catch this exception instead?

```
try {
  ...
} catch (org.jenkinsci.plugins.workflow.steps.FlowInterruptedException fie) {
  ...
} catch (e) {
  ...
}
```

Reply:
No idea what I was thinking doing that.. have fixed it, but it bought more complication.

Reply:
Ugh!